asserts/export_test.go

```diff
@@ -28,8 +28,14 @@ import (
 
 // expose test-only things here
 
-// generatePrivateKey exposed for tests
-var GeneratePrivateKeyInTest = generatePrivateKey
+// XXX: just use GenerateKey
```

Comment:
Yeah, we need to clean this up, but let's talk about that design first so it's more clear how to do that.